docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,471 @@
+
+Securing event delivery in Knative Eventing is essential to prevent unauthorized access. To enforce fine-grained control over event delivery, Knative Eventing introduces the `EventPolicy` custom resource, which allows users to specify which entities are authorized to send events to specific consumers within a namespace.
+
+## Prerequisites
+
+- [Eventing installation](./../../../install)
+- [`authentication-oidc`](./sender-identity.md) feature to be enabled.
+
+!!! note
+    As described on the [`authentication-oidc`](./sender-identity.md), `transport-encryption` should be enabled as well for a secure authentication. Take a look at [Transport-Encryption](./transport-encryption.md), which explains how to enable the transport encryption feature flag.
+
+## Compatibility
+
+Authorization is currently supported for the following components:
+
+- Brokers:
+    - [MTChannelBasedBroker](./../brokers/broker-types/channel-based-broker/)
+    - [Knative Broker for Apache Kafka](./../brokers/broker-types/kafka-broker/)
+- Channels:
+    - [InMemoryChannel](./../channels/channels-crds.md)
+    - [KafkaChannel](./../channels/channels-crds.md)
+- Flows:
+    - [Sequence](./../flows/sequence/README.md)
+    - [Parallel](./../flows/parallel.md)
+- Others:
+    - [KafkaSink](./../sinks/kafka-sink.md)
+
+## Default Authorization Mode
+
+Administrators can set a default authorization mode using the `defaultAuthorizationMode` feature flag, which Knative Eventing will use whenever no `EventPolicy` applies to a resource. The available modes are:
+
+* `allow-all`: All requests are allowed.
+* `deny-all`: All requests are denied, enforcing the creation of EventPolicies.
+* `allow-same-namespace`: Only requests from subjects within the same namespace are allowed. (Default)
+
+## Defining an EventPolicy
+
+An `EventPolicy` defines rules for event delivery by specifying which subjects (service accounts or event sources) are allowed to send events to designated event consumers.
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: EventPolicy
+metadata:
+  name: my-event-policy
+  namespace: default
+spec:
+  to:
+    - ref:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        name: my-broker
+    - selector:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        matchLabels:
+          app: special-app
+  from:
+    - ref:
+        apiVersion: sources.knative.dev/v1
+        kind: PingSource
+        name: my-source
+        namespace: another-namespace
+    - sub: system:serviceaccount:default:trusted-app
+    - sub: "system:serviceaccount:default:other-*"
+  filters:
+    - cesql: "type IN ('order.created', 'order.updated', 'order.canceled')"
+    - exact:
+        type: com.github.push
+```
+
+### Specify for who the `EventPolicy` applies
+
+The `.spec.to` section specifies **where** the events are allowed to be sent. This field is optional; if left empty, the policy applies to all resources within the namespace.
```

Comment:
Is the `to.*` fields logically combined in an `or` statement? or an `and` statement?  I'm not sure from the description if specifying multiple `to.*` targets widens the scope of the event policy, or narrows it.

Reply:
Interesting point of view to this. Let me try to make it clearer. 
For background information: AuthZ is checked on the receiving side, so we don't know at this point, where the events got sent to elsewhere.

Reply:
Updated. Does this make it clearer?